.travis.yml

```diff
@@ -7,7 +7,6 @@ node_js:
 - '6.1'
 env:
   matrix:
-  - TEST_BROWSER_NAME=PhantomJS
```

Comment:
This is a bit of an issue as it makes it difficult to run locally. What's preventing us from using PhantomJS 1.5?